redis/reply.go

```diff
@@ -479,6 +479,55 @@ func Positions(result interface{}, err error) ([]*[2]float64, error) {
 	return positions, nil
 }
 
+// Same as Int64s, Uint64s is a helper that converts an array command reply to a []uint64.
+// If err is not equal to nil, then Uint64s returns nil, err. Nil array
+// items are stay nil. Uint64s returns an error if an array item is not a
+// bulk string or nil.
+func Uint64s(reply interface{}, err error) ([]uint64, error) {
+	var result []uint64
+	err = sliceHelper(reply, err, "Uint64s", func(n int) { result = make([]uint64, n) }, func(i int, v interface{}) error {
+		switch v := v.(type) {
+		case uint64:
+			result[i] = v
+			return nil
+		case []byte:
+			n, err := strconv.ParseUint(string(v), 10, 64)
+			result[i] = n
+			return err
+		default:
+			return fmt.Errorf("redigo: unexpected element type for Uint64s, got type %T", v)
+		}
+	})
+	return result, err
+}
+
+
+// Same as Int64Map , Uint64Map is a helper that return a  map[uint64]uint64 data. 
```

Comment:
I think this one will lead to confusion, I would expect `map[string]uint64` not `map[uint64]uint64` as `Uint64Map` does.

This feels like more of a special case than we should include in this library if this was the real intention?

As per above no need to refer to the `Same as...` and this isn't the format `HGETALL` returns as that's strings, sure it could work but for the most part not, so this could lead to confusion.

Also missing a test for this method.

Reply:
In projects, sometimes I really want to return map[uint64][uint64]. 
maybe I should change the func name `Uint64Map` ?
Or do you have any other suggestions？

Reply:
You may yes but I would say its much less common to have field name that's an integer, I can't say we have any usages for that here.

Reply:
> You may yes but I would say its much less common to have field name that's an integer, I can't say we have any usages for that here.
> 
> So I would change this to map[string]uint64

ok, got it.